searx/static/themes/simple/package.json

```diff
@@ -9,17 +9,17 @@
     "grunt-contrib-less": "~3.0.0",
     "grunt-contrib-uglify": "~5.0.1",
     "grunt-xmlmin": "~0.1.8",
-    "grunt-contrib-htmlmin": "~3.1.0",
     "grunt-contrib-watch": "~1.1.0",
     "grunt-eslint": "^23.0.0",
     "grunt-stylelint": "^0.16.0",
+    "grunt-image": "6.x",
     "ionicons": "^5.5.4",
     "less": "^4.1.1",
     "less-plugin-clean-css": "^1.5.1",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^22.0.0",
     "ejs": "^3.1.6",
-    "svgo": "^2.8.0"
+    "svgo": "2.x"
```

Comment:
@dalf @MrPaulBlack I used the `1.x` and `1.0.x` markup for the [semantic versioning](https://docs.npmjs.com/about-semantic-versioning).

I don't know what you prefer or what is more common, but for me the `x` placeholder is more intuitive compared to `^` and `~`.

Reply:
I have overlooked these options.
On https://semver.npmjs.com/ when I try 
* lodash
* ^1.1.0

The results are `1.1.x`, `1.2.x`, `1.3.x`.

I don't know how to do with the `1.1.x` notation.

Reply:
>  How to write the same requirement (`^1.1.0`) with the `1.1.x` notation ?

just use `1.1.x`

- `1.x` is equal to `^1.0.0`
- `1.1.x` is equal to `~1.1.0`

That is why I think `n.x` and `n.n.x` is more intuitive otherwise you need to know for what `~` and `^` are for / See https://docs.npmjs.com/about-semantic-versioning

```
    Patch releases: 1.0 or 1.0.x or ~1.0.4
    Minor releases: 1 or 1.x or ^1.0.4
    Major releases: * or x
```

Reply:
After thinking twice I think this notation is most intuitive: 

    1.1 --> patch releases
    1 --> minor releases
    * --> major releases

Reply:
> just use 1.x

AFIK, nope, `^1.1.0` means `1.1.0` or all versions above but below `2.x`. You can't write that with the `x` notation.

Reply:
Hm, I am a bit confused, seems you are right about the selectors ..

> AFIK, nope, `^1.1.0` means `1.1.0` or all versions above but below `2.x`. You can't write that with the `x` notation.

... but would a `1.1.0` be installed when a `1.1.1`or `1.2.0` exists?

Reply:
> would a 1.1.0 be installed when a 1.1.1or 1.2.0 exists?

If the version 1.x is required but the 1.1.0 version fixes an issue.

Reply:
> If the version 1.x is required but the 1.1.0 version fixes an issue.

Hm .. I do not understand this answer .. when `1.1.0` exist, it fixes an issue of the previous minor version.

My question is, where is the difference in the result of what version is installed when I use `^1.1.0` compared to `1.x` (or `1`) .. al three notation will install the same package.

> `^1.1.0` means `1.1.0` or all versions above but below `2.x`.

So why should I select ^1.1.0`` when `1` (or `1.x`) ends in the same package version that is installed?

Reply:
I changed back to the caret (aka hat) symbol / see https://github.com/searxng/searxng/pull/503/commits/b07884c95864a074b700d635c92a43f734cc8868#diff-0f087f6a54013c2b6e21074952c187930cd193591ec780748f18423271435f90

X-Ranges : https://github.com/npm/node-semver#x-ranges-12x-1x-12-
Caret Ranges: https://github.com/npm/node-semver#caret-ranges-123-025-004